packages/backend/src/controllers/commentsController.ts

```diff
@@ -0,0 +1,139 @@
+import { ApiErrorPayload, Comment } from '@lightdash/common';
+import {
+    ApiCreateComment,
+    ApiGetComments,
+    ApiResolveComment,
+} from '@lightdash/common/src/types/api/comments';
+import {
+    Body,
+    Controller,
+    Delete,
+    Get,
+    Middlewares,
+    OperationId,
+    Patch,
+    Path,
+    Post,
+    Request,
+    Response,
+    Route,
+    SuccessResponse,
+    Tags,
+} from '@tsoa/runtime';
+import express from 'express';
+import { dashboardService } from '../services/services';
+import { allowApiKeyAuthentication, isAuthenticated } from './authentication';
+
+@Route('/api/v1/comments')
+@Response<ApiErrorPayload>('default', 'Error')
+@Tags('Comments')
+export class CommentsController extends Controller {
+    /**
+     * Creates a comment on a dashboard tile
+     * @param req express request
+     * @param body the comment to create
+     * @param dashboardUuid the uuid of the dashboard
+     * @param dashboardTileUuid the uuid of the dashboard tile
+     * @returns the id of the created comment
+     */
+    @Middlewares([allowApiKeyAuthentication, isAuthenticated])
+    @SuccessResponse('200', 'Success')
+    @Post('/dashboards/{dashboardUuid}/{dashboardTileUuid}')
```

Comment:
would you declare this endpoint differently to avoid resource nesting @rephus ?

Reply:
we can do `/api/v1/comments/dashboards/{dashboardUuid}/{dashboardTileUuid}`  so don't need to change that, only main Route 

question, if we want to show this comments directly on the chart, how can we link this tileuuid to the chartuuid ? Should we save teh chartuuid (nullable) on the comment too ?

Reply:
Ohhhh, guess we didn't think about that yet... 
I guess so! Should we add that on a separate PR?

Reply:
this can be aseparate PR, but I'd like to minimize the number of migrations if possible. So I'd add the `chart_uuid` on this PR, and start populating the field, so we can start using this in charts efficiently if we want to.